examples/howto/marginalizing-models.ipynb

```diff
@@ -0,0 +1,1510 @@
+{
```

Comment:
<p>Some typo somewhere</p><p>in </p><p> </p><p>"to where that choice is not our model. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/628/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>